core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -317,169 +317,272 @@ protected boolean removeEldestEntry(Map.Entry<Sha256Hash, Transaction> eldest) {
      * Creates a new, empty wallet with a randomly chosen seed and no transactions. Make sure to provide for sufficient
      * backup! Any keys will be derived from the seed. If you want to restore a wallet from disk instead, see
      * {@link #loadFromFile}.
-     * @param params network parameters
+     * @param network network
```

Comment:
This would be a good opportunity to come up with a better param description and use it here and in all the other new constructors.

Reply:
Fixed.